src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
```

Comment:
The class `IsoscelesTrapezoid` should not begin with an empty line. Please remove the empty line to follow the Java class structure conventions.